x-pack/legacy/plugins/siem/server/lib/detection_engine/routes/schemas/patch_rules_bulk_schema.test.ts

```diff
@@ -49,4 +49,41 @@ describe('patch_rules_bulk_schema', () => {
       ]).error
     ).toBeFalsy();
   });
+
+  test('can set "note" to be a string', () => {
+    expect(
+      patchRulesBulkSchema.validate<Array<Partial<PatchRuleAlertParamsRest>>>([
+        {
+          id: 'rule-1',
+          note: 'hi',
+        },
+      ]).error
+    ).toBeFalsy();
+  });
+
+  test('can set "note" to be an empty string', () => {
+    expect(
+      patchRulesBulkSchema.validate<Array<Partial<PatchRuleAlertParamsRest>>>([
+        {
+          id: 'rule-1',
+          note: '',
+        },
+      ]).error
+    ).toBeFalsy();
+  });
+
+  test('cannot set "note" to be anything other than a string', () => {
+    expect(
+      patchRulesBulkSchema.validate<
+        Array<Partial<Omit<PatchRuleAlertParamsRest, 'note'> & { note: object }>>
+      >([
+        {
+          id: 'rule-1',
+          note: {
+            someprop: 'some value here',
+          },
+        },
+      ]).error
+    ).toBeTruthy();
```

Comment:
Using a string value rather than truthy will avoid false positives in the future.

For example:

```ts
      ]).error.message
    ).toEqual(
      '"value" at position 0 fails because [child "note" fails because ["note" must be a string]]'
    );
```

You can see that it explicitly is true because of the string given to it.

Reply:
Thanks for pointing that out. I've gone ahead and updated the test.